src/active_window/reconstruction_module.cpp

```diff
@@ -156,6 +159,17 @@ ActiveWindowOutput::Ptr ReconstructionModule::spinOnce(const InputPacket& msg) {
   cv::Mat integration_mask;
   maskInvalidSemantics(data->label_image, invalid_labels, integration_mask);
 
+  cv::Mat static_mask;
+  if (config.tsdf.static_mask != ""){
+    std::string mask_path = ros::package::getPath("hydra") + "/config/static_masks/" + config.tsdf.static_mask + ".png";
```

Comment:
We don't want to force the path to be relative to hydra -- you can assume that the provided path from the config is absolute, and any relative path logic like this can be handled in the launch/config file.

Reply:
Ah super helpful. That was a part I was struggling a bit with simply because ROS1 and ROS2 handle this differently.